### PR TITLE
add diher.solutions and rss.my.id to the list

### PR DESCRIPTION
**Pull request by pereceh**
add a new public suffix for Diher Solutions to use by yourself. everyone can use the domains I added for informational purposes.

* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the _PSL txt record in place

__Submitter affirms the following:__ 
  * [x] We are listing any third party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
  * [x] This request was _not_ submitted with the objective of working around other third party limits
  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting

---
__For Private section requests that are submitting entries for domains that match their organization website's primary domain:__

``` 
Seriously, carefully read the downline flow of the PSL and the guidelines.
Your request could very likely alter the cookie and certificate (as well as other) behaviours on your 
core domain name in ways that could be problematic for your business.

Rollback is really not predicatable, as those who use or incorporate the PSL do what they do, and when.
It is not within the PSL volunteers' control to do anything about that.  

The volunteers are busy with new requests, and rollbacks are lowest priority, so if something gets broken 
it will stay that way for an indefinitely long while.
```
(Link: [about propogation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] Yes, I understand.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  Proceed.
---

Description of Organization
====

Organization Website:  https://rss.my.id

I am a blog writer. and I'm also the sole manager for the listed domains.

Reason for PSL Inclusion
====

I'm planning to automate the provision of a destination subdomain in place of an address on a web hosting server. my purpose of implementing this is for Cookie Security, as different customers will have control over the subdomain.

The rss.my.id domain is on a 3-year renewal and currently registered until 2024-07-24,

and diher.solutions domain is on a 9-year renewal and currently registered until 2031-03-15.

DNS Verification via dig
=======

```
dig +short TXT _psl.diher.solutions
"https://github.com/publicsuffix/list/pull/1393"
```

```
dig +short TXT _psl.rss.my.id
"https://github.com/publicsuffix/list/pull/1393"
```
make test
=========

All tests passed.

### CODE REVIEW

**Comment by pereceh** [comment]
I am the owner of this domain. I made this for informational purposes only, nothing more.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -13760,4 +13760,11 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// Diher Solutions : https://diher.solutions
```
Comment (dnsguru) [inline public_suffix_list.dat:13763]: Need to move the added section up to a higher place in the PRIVATE DOMAINS section, sorted ascending alphabetically by your org name in order to comply with guidelines

File: public_suffix_list.dat [hunk 2]
```diff
@@ -13760,4 +13760,11 @@ basicserver.io
...
+// Submitted by Didi Hermawan <didihermawan@aol.com>
+*.diher.solutions
```
Comment (dnsguru) [inline public_suffix_list.dat:13765]: Need to sort the entries by TLD then SLD in Ascending Alphabetical order (id would come before solutions) in order to comply with the guidelines

**Comment by dnsguru** [comment]
Made some suggestions about the contributor guidelines and ordering of the submitted PR.  Needs updating to meet the guidelines.

Also: The AOL email address bounced as undeliverable on the contact in the header.  Is there an alternative company / submitter name that could be used that has better reachability?

**Comment by pereceh** [comment]
> Made some suggestions about the contributor guidelines and ordering of the submitted PR. Needs updating to meet the guidelines.
> 
> Also: The AOL email address bounced as undeliverable on the contact in the header. Is there an alternative company / submitter name that could be used that has better reachability?


do i need to open whois privacy ?

File: public_suffix_list.dat (hunk 1 above)
Comment (pereceh) [reply public_suffix_list.dat]: fixed

**Review by dnsguru — CHANGES_REQUESTED** [review]
no comment on whois privacy.  my comment was that there was an attempt to email that listed contact which was not possible.  That's not wise to have at the onset, especially when there seems to be so much iteration required to be involved, and the recommendations had been articulated in the bounced email.  We definitely require a valid email address to be present, so perhaps consider a reachable replacement for the AOL address as you're making these other changes.

Two changes below.  If we start to have further challenges in getting this filled out correctly, it may be time to request you get help from someone on filling this out.

File: public_suffix_list.dat [hunk 3]
```diff
@@ -13760,4 +13767,5 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+
```
[the remark this answers is not in the document]
Comment (pereceh) [reply public_suffix_list.dat]: done

File: public_suffix_list.dat [hunk 4]
```diff
@@ -11246,6 +11246,13 @@ deno-staging.dev
 // Submitted by Peter Thomassen <peter@desec.io>
 dedyn.io
 
+// Diher Solutions : https://diher.solutions
+// Submitted by Didi Hermawan <mail@diher.solutions>
+diher.solutions
+*.diher.solutions
+rss.my.id
```
[the remark this answers is not in the document]
Comment (pereceh) [reply public_suffix_list.dat:11251]: done

**Comment by pereceh** [comment]
> no comment on whois privacy. my comment was that there was an attempt to email that listed contact which was not possible. That's not wise to have at the onset, especially when there seems to be so much iteration required to be involved, and the recommendations had been articulated in the bounced email. We definitely require a valid email address to be present, so perhaps consider a reachable replacement for the AOL address as you're making these other changes.
> 
> Two changes below. If we start to have further challenges in getting this filled out correctly, it may be time to request you get help from someone on filling this out.

that's good. i have fixed it. thank you for guiding me. please check again.

**Review by dnsguru — APPROVED** [review]
- DNS Validation ok
- Tests ok
- Rationale ok
- No conflict with base
- Formatted / Sorted per Guidelines

**Approved**

**Comment by dnsguru** [comment]
Thank you for remedying the sorting.  Merging.  I see above where you acknowledged reading the understanding and acceptance of [propogation](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion).  You are affecting your core URL for diher.solutions, so ... good luck.

**Comment by rvandermeulen** [comment]
@dnsguru Does it make sense to be adding both *.diher.solutions and diher.solutions to the list? At least for Mozilla, this is breaking when we try to update PSL data for Firefox due to repeating domain name errors.

**Comment by sleevi** [comment]
@rvandermeulen Yeah, that will also break Chrome.

Our linter was supposed to catch this - https://github.com/publicsuffix/list/blob/master/linter/test_duplicate.input

We had the linter wired up to Travis in https://github.com/publicsuffix/list/commit/e6065d27b4ef9bd74b7d430775f2f6233d4ff581#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485 (and refactored subsequently into the Makefile), but we never wired that up for GitHub Actions, and the whole Travis kerfuffle has meant we've lost testing coverage.

Any chance you'd be willing to fiddle with GitHub Actions to get it running our lints again? I'm not sure I'll be able to get to it in the next week, but that would help prevent this for both you (and us/Chrome)

**Comment by dnsguru** [comment]
Interesting ... we need to require binary choice between the two for this.

Needs a rollback pr I will do now

**Comment by sleevi** [comment]
Yeah, that's captured in https://github.com/publicsuffix/list/issues/694#issuecomment-406591178

**Comment by dnsguru** [comment]
@sleevi can you review #1406

**Pull request merged**